ister.py

```diff
@@ -412,9 +412,17 @@ def get_current_format():
 def copy_os(args, template, target_dir):
     """Wrapper for running install command
     """
-    LOG.info("Starting swupd. May take several minutes")
-    add_bundles(template, target_dir)
+    package_manager = template["SoftwareManager"]
+    LOG.info("Starting {0}. May take several minutes".format(package_manager))
+    if package_manager == "swupd":
+        copy_os_swupd(args, template, target_dir)
+    elif package_manager == "dnf":
+        copy_os_dnf(args, template, target_dir)
+
 
+def copy_os_swupd(args, template, target_dir):
```

Comment:
Needs unit test.

Reply:
Could the existing unit tests be used here because we're defaulting to swupd? Or, maybe I should add a unit test for checking the new copy_os function works to choose the correct copy_os_xxx as expected?

Reply:
The latter, we want to ensure the right one is picked, can stub out the actual call to copy_os_xxx in the test.